open-company/open-company-web
# Linked charts

Card: https://trello.com/c/ZWMu5BHt

Related PRs:
- [lib](https://github.com/open-company/open-company-lib/pull/7)
- [proxy](https://github.com/open-company/open-company-proxy): no PR, but mainline review mainline code
- [infrastructure](https://github.com/belucid/open-company-infrastructure/pull/37)

To test:
- add a chart using the new UI
- save and check that the chart is rendered as expected
- go back to the google sheet and update the chart
- check that the chart doesn't change on our side
- edit the topic and click the refresh button
- check that the chart is updated now
- now if you click SAVE the new chart should be kept, if you CANCEL the old one should re-appear (at least until the cache doesn't expire)
- remove the chart and save
- if everything is working as expected merge!
